pom.xml

```diff
@@ -56,7 +56,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>credentials</artifactId>
-            <version>1.22</version>
+            <version>2.1.6</version>
```

Comment:
Well needs to be at least 2.1.0 based on transitive dependencies, and probably should be 2.1.5 for big fixes... and 2.1.6 is zero risk fix on 2.1.5... so hence 2.1.6